### PR TITLE
Ellsworth_2015: add new study

### PR DESCRIPTION
**Pull request by ehwenk**
* add new study, Ellsworth_2015
* report reviewed by contributor

### CODE REVIEW

File: data/Ellsworth_2015/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,1225 @@
...
+    from 1 m up to 25 m high (Table 1). For tall species, access to the upper parts
+    of the tree crowns was achieved by three different means, an articulated boom
+    lift (Snorkel MHP13/35 Trailer Mounted Lift; Snorkel Ltd., Meadowbrook, Qld, Australia)
+    used at the Hawkesbury site in Richmond, NSW; a set of 36 m tall construction
+    cranes (Jaso crane J-4010; Jaso S.L., Idiazabal, Spain) at the nearby EucFACE
+    site in Richmond, NSW; and a custom-built steel-alloy canopy walkway going up
+    from ground level to 30 m height ('Illawarra fly') at Robertson, NSW. Canopy access
+    was not necessary at the Lesueur National Park site or at Davies Park, as trees
+    and shrubs were open-grown in each of the sclerophyll woodlands, and unshaded
+    leaves at the outside of the crown could be readily measured.
```
Comment (yangsophieee) [inline data/Ellsworth_2015/metadata.yml:912]: In the readme file for this data, they say they measured LMA without petiole, so does that mean there should be a `measurement_remarks column` mentioning this?

**Review by yangsophieee — CHANGES_REQUESTED** [review]
Looking great! Just a few trivial questions-- see comments 😊

File: data/Ellsworth_2015/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,1225 @@
...
+    slight over-pressure as for ambient air as described by Li-Cor (Li-Cor 2008) and
+    with the excess flow to the Li-6400 pump monitored with a rotameter. A Teflon
+    T-valve was toggled between ambient air with 21 kPa pO2 and 2 kPa tank gas at
+    the appropriate Ca steps (up to five Ca steps including at saturation). These
+    steps were chosen in order to minimally define the initial rise to a maximum and
+    the maximum asymptote for the Anet-Cc curve at low pO2, given that the shape of
+    these curves has long been known (Laing et al. 1974; von Caemmerer 2000). The
+    flow excess was maintained around 0.3 L min-1. Measurements of Anet in 2 kPa pO2
+    were completely reversible as described in Laing et al. (1974) (see Supporting
+    Information Fig. S1).
```
[the remark this answers is not in the document]
Comment (ehwenk) [reply data/Ellsworth_2015/metadata.yml:402]: it is, but it isn't really a trait, so it is being ignored.

File: data/Ellsworth_2015/metadata.yml [hunk 3]
```diff
@@ -0,0 +1,1225 @@
...
+    for species level Vcmax, Jmax and Tp at both pO2 levels using the 'nls' package
+    in R. Across species, the two sets of solutions agreed well with one another,
+    since slopes for each parameter were close to unity (slopes of 0.981, 0.966 and
+    0.840 for Vcmax, Jmax and Tp, respectively, estimated for piecewise compared with
+    simultaneously solved). Vomax, the maximum velocity of oxygenase activity, was
+    fited to the data from both pO2 levels for low Cc where oxygenase activity of
+    Rubisco is considered limiting, following equations in Farquhar et al. (1980).
+- var_in: Rd
+  unit_in: umol{CO2}/m2/s
+  trait_name: .na
```
[the remark this answers is not in the document]
Comment (ehwenk) [reply data/Ellsworth_2015/metadata.yml:709]: yes, intentional

File: data/Ellsworth_2015/metadata.yml (hunk 1 above)
Comment (ehwenk) [reply data/Ellsworth_2015/metadata.yml:912]: I'll add that.

Comment (ehwenk) [reply data/Ellsworth_2015/metadata.yml:912]: fixed

File: data/Ellsworth_2015/metadata.yml [hunk 4]
```diff
@@ -0,0 +1,1225 @@
...
+    ORCID: 0000-0002-3178-0674
+    affiliation: The Open University, Milton Keynes, United Kingdom
+  austraits_curators: Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code:   '
+    data %>%
+      mutate(Species = stringr::str_replace(Species, "_", " "))
+  '
+  collection_date: Date
```
[the remark this answers is not in the document]
Comment (ehwenk) [reply data/Ellsworth_2015/metadata.yml:42]: fixed

**Review by yangsophieee — APPROVED** [review]

**Comment by dfalster** [comment]
Nice work! I noticed ancient Co2 listed in metadata as a trait. Should this be context?

**Comment by ehwenk** [comment]
ambient CO2? Yes, but I am going to wait until we go the next step of not having to list all the CO2 values in the metadata file to reduce the amount of "clutter" (issue #673)

**Pull request merged**